cpplib/Tests/GeometryTest.cpp

```
@@ -831,164 +829,296 @@ TEST_F(VoronoiTest, AlexTest) {
-    cells = vdWithFlags.extractCells();
-    EXPECT_EQ(cells.size(), singlePoint.size());
+		ASSERT_EQ(a, alive[i][0]);
+		ASSERT_EQ(b, alive[i][1]);
+		ASSERT_EQ(c, alive[i][2]);
+	}
+	voronoi::VoronoiFused vf(cells);
+	ASSERT_EQ(vf.vertices.size(), 281);
+	ASSERT_EQ(vf.edges.size(), 475);
+	ASSERT_EQ(vf.polygons.size(), 212);
 }
+TEST_F(VoronoiTest, Benzene) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)